visma/io/tokenize.py

```diff
@@ -331,10 +344,22 @@ def normalize(terms):
     Returns:
         terms {list} -- Greek input terms
     """
-    for term in terms:
+    for index, term in enumerate(terms):
         for i, x in enumerate(inputLaTeX):
             if x == term:
                 term = inputGreek[i]
+        if term == 'frac':
```

Comment:
Currently crashes when input is given byte by byte. To fix this you can modify _checkEquation_ function in visma/io/checks.py. Add a case for '$' and an error expression (for ex: "LaTeX detected: Missing ending $").